packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -1063,13 +898,7 @@ export class _GLTFMaterialExporter {
         return pixels;
     }
 
-    /**
-     * Extracts a texture from a Babylon texture into file data and glTF data
-     * @param babylonTexture Babylon texture to extract
-     * @param mimeType Mime Type of the babylonTexture
-     * @returns glTF texture info, or null if the texture format is not supported
-     */
-    public _exportTextureAsync(babylonTexture: BaseTexture, mimeType: ImageMimeType): Promise<Nullable<ITextureInfo>> {
+    public async _exportTextureAsync(babylonTexture: BaseTexture, mimeType: ImageMimeType): Promise<Nullable<ITextureInfo>> {

```

Comment:
Was this meant to be private?

Reply:
Initially yes, but this end up been used in KHR_materials_transmission.